assets/javascripts/utils.js

```diff
@@ -27,3 +27,29 @@ export const capitalizeFirstLetter = (string) => {
 export const convertNodeListToArray = (nl) => {
   return Array.prototype.slice.call(nl)
 }
+
+export const makePreventScrollFn = (scrollDivClass = []) => {
```

Comment:
At some point we might look at this name and see if we can find a better one :-)